Source/Scene/Model.js

```diff
@@ -2481,14 +2483,22 @@ define([
     }
 
     function getChannelEvaluator(model, runtimeNode, targetPath, spline) {
+        if (model._clampAnimations) {
+            return function(localAnimationTime) {
+                if (defined(spline)) {
+                    runtimeNode[targetPath] = spline.evaluate(spline.clampTime(localAnimationTime), runtimeNode[targetPath]);
+                    runtimeNode.dirtyNumber = model._maxDirtyNumber;
```

Comment:
Should the two cases be an if-else in the same function? That way someone would be able to change the `clampAnimation` property dynamically. I don't think there would be much of a performance hit. `this._clampAnimations` could then become `this.clampAnimations` and be a public property.